src/main/java/com/example/springbootbasic/util/CharacterUnit.java

```diff
@@ -0,0 +1,19 @@
+package com.example.springbootbasic.util;
+
+public enum CharacterUnit {
+    SPACE(" "),
+    ENTER("\n"),
```

Comment:
```suggestion
    ENTER(System.lineSeparator()),
```

의견 나왔던 것처럼 os 상관없이 사용할 수 있는 것으로 사용해도 좋겠네요.

Reply:
<<수정했습니다>>

System.lineSeparator()를 사용하면
실행하는 OS의 개행 문자로 리턴해서 더 정확할 수 있네요 👍

[OS 실행환경에 따른 개행 문자 수정](https://github.com/hyena0608/spb/commit/f597995a6f9d8bbf886ebb9412a2e957cc484571)